README.md

```diff
@@ -18,10 +18,9 @@ We are honored by any contributions you may have small or large. Please refer to
 
 The following Java and Node combinations are tested and verified by GitHub Actions:
 
-| Java | Node | Status |
-| ---- | ---- | ------ |
-| 21   | 20   | ✅     |
-| 17   | 18   | ✅     |
+|   Java   |  Node   | Status |
+| -------- | ------- | ------ |
+| 17/21/22 |  18/20  |   ✅   |
```

Comment:
Would it be accurate to say 17-21 and 18-20 or do you think it's better to specify the specific versions like you have here?

Reply:
I think the old table not accurate, if looks like java 21 is tested with node 20 and java 17 with node 18.
We do have tests in place for java 17/21/22, not 18/19/20. Same for node, so specific versions seems better .